lineman/app/components/email_settings_page/email_settings_page_controller.coffee

```diff
@@ -0,0 +1,12 @@
+angular.module('loomioApp').controller 'EmailSettingsPageController', (Records, FlashService, CurrentUser, $location) ->
+  @init = ->
+    @user = CurrentUser
```

Comment:
Hm, I'm thinking I may have set a poor precedent doing this init() thing on the profile page; I only did that in order to ensure that the profile picture updated correctly. I don't think you need it here, you can just do

```
@user = CurrentUser.clone()
```

(ps I believe we want to `clone()` the CurrentUser here, so the settings aren't updated before the form is submitted.)